### PR TITLE
Updated publications and presentations

### PR DESCRIPTION
**Pull request by smgallo**
## Description

Updated publications and presentations to include 2016 presentations and PEARC17 accepted publications.

## Motivation and Context

Keep information up-to-date.

## Tests performed

Viewed in browser.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project as found in the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,42 +2,44 @@
...
+of application performance to aid in optimizing code performance, (4)
+provide stakeholders with a diagnostic tool to facilitate HPC planning
+and analysis, and (5) provide metrics to help measure scientific impact.
+In addition, analyses of the operational characteristics of the HPC
+environment can be carried out at different levels of granularity,
+including job, user, or on a system-wide basis.
+
+The Open XDMoD portal provides a rich set of features accessible through
+an intuitive graphical interface, which is tailored to the role of the
+user.  Metrics provided include: number of jobs, CPUs consumed, wait
```
Comment (jpwhite4) [inline docs/index.md]: I don't like the phrase "CPUs consumed"

Suggest either "job size" or "CPU hours consumed" depending on what the original phrase was supposed to mean.

Comment (smgallo) [inline docs/index.md]: Oops, looks like this pulled in Jeff's modification to the 6.6 branch as well after I git a `git pull`. That's the text that Tom wanted updated...

Comment (jtpalmer) [inline docs/index.md]: I've emailed Tom to get his opinion.

Comment (jtpalmer) [inline docs/index.md]: I've created #135 to address this issue.  You can rebase this branch against xdmod6.7 and remove those commits.

**Pull request closed without merging**